client.lua

```diff
@@ -535,6 +563,8 @@ RegisterNetEvent('qb-vehicleshop:client:openCustomFinance', function(data)
         data.vehicle, playerid)
 end)
 
+--- Swaps the chosen vehicle with another one
+---@param data table<ClosestVehicle, toVehicle>
```

Comment:
No idea what to do here.

Reply:
the table<key, value> type is used for maps, to specify the key and value type of the table. In this case from context the keys of the data table are literally "ClosestShop", "ClosestVehicle", and "toVehicle". Since the data table has literal keys and is what I would consider an object (in the JavaScript/JSON definition of object), a table literal would be the appropriate type https://github.com/LuaLS/lua-language-server/wiki/Annotations#documenting-types